ui/src/app/shared/models.ts

```diff
@@ -586,6 +586,13 @@ export interface HelmChart {
 
 export type AppSourceType = 'Helm' | 'Kustomize' | 'Directory' | 'Plugin';
 
+export enum AppSource {
+    Helm = 'Helm',
+    Kustomize = 'Kustomize',
+    Directory = 'Directory',
+    Plugin = 'Plugin'
+}
+
```

Comment:
I couldn't locate any mention of this enum anywhere. It seems safe to remove it.

Reply:
It's being used in application-parameters.tsx. But since it is only used there, I'll just get rid of it.